controllers/nginx/pkg/template/configmap.go

```diff
@@ -73,12 +79,27 @@ func ReadConfig(src map[string]string) config.Configuration {
 	} else {
 		proxylist = append(proxylist, "0.0.0.0/0")
 	}
+	if val, ok := conf[bindAddress]; ok {
+		delete(conf, bindAddress)
+		for _, i := range strings.Split(val, ",") {
+			ns := net.ParseIP(i)
+			if ns != nil {
```

Comment:
If the IP is not valid,`net.ParseIP(i)` returned nil, then we should log a warning specifying which value is wrong. That way the IP will fail to be added to the list gracefully and the user will know what is the cause of any unexpected behavior.